### PR TITLE
Update run statuses

### PR DESCRIPTION
**Pull request by rberlind**

### CODE REVIEW

**Comment by nfagerlund** [comment]
@rberlind Thanks for checking up on this. Could I get a citation link to confirm (either to the atlas repo, or to an asana ticket or slack thread with discussion)?

**Comment by rberlind** [comment]
Sorry, I had given the citation on the commits, but not on the PR:
I based this on the list in https://github.com/hashicorp/atlas/blob/master/app/models-new/run.rb

File: content/source/docs/cloud/api/admin/workspaces.html.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Status  | Response                                     | Reason
 Parameter                     | Description
 ------------------------------|------------
 `q`                           | **Optional.** A search query string. Workspaces are searchable by name and organization name.
-`filter[current_run][status]` | **Optional.** A comma-separated list of Run statuses to restrict results to, including of any of the following: `"pending"`, `"planning"`, `"planned"`, `"confirmed"`, `"applying"`, `"applied"`, `"discarded"`, `"errored"`, `"canceled"`, `"policy_checking"`, `"policy_override"`, and/or `"policy_checked"`.
+`filter[current_run][status]` | **Optional.** A comma-separated list of Run statuses to restrict results to, including the following: `"pending"`, `"plan_queued"`, `"planning"`, `"planned"`, `"confirmed"`, `"apply_queued"`, `"applying"`, `"applied"`, `"discarded"`, `"errored"`, `"canceled"`, `"cost_estimating"`, `"cost_estimated"`, `"policy_checking"`, `"policy_override"`, `"policy_soft_failed"`, `"policy_checked"`, and `"planned_and_finished"`.
```
Comment (nfagerlund) [inline content/source/docs/cloud/api/admin/workspaces.html.md]:
```suggestion
`filter[current_run][status]` | **Optional.** A comma-separated list of Run statuses to restrict results to, which can include any of the following: `"pending"`, `"plan_queued"`, `"planning"`, `"planned"`, `"confirmed"`, `"apply_queued"`, `"applying"`, `"applied"`, `"discarded"`, `"errored"`, `"canceled"`, `"cost_estimating"`, `"cost_estimated"`, `"policy_checking"`, `"policy_override"`, `"policy_soft_failed"`, `"policy_checked"`, and `"planned_and_finished"`.
```

File: content/source/docs/cloud/api/admin/runs.html.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ Status  | Response                               | Reason
 Parameter           | Description
 --------------------|------------
 `q`                 | **Optional.** A search query string. Runs are searchable by ID, workspace name, organization name or email, and VCS repository identifier.
-`filter[status]`    | **Optional.** A comma-separated list of Run statuses to restrict results to, including any of the following: `"pending"`, `"planning"`, `"planned"`, `"confirmed"`, `"applying"`, `"applied"`, `"discarded"`, `"errored"`, `"canceled"`, `"policy_checking"`, `"policy_override"`, and/or `"policy_checked"`.
+`filter[status]`    | **Optional.** A comma-separated list of Run statuses to restrict results to, including the following: `"pending"`, `"plan_queued"`, `"planning"`, `"planned"`, `"confirmed"`, `"apply_queued"`, `"applying"`, `"applied"`, `"discarded"`, `"errored"`, `"canceled"`, `"cost_estimating"`, `"cost_estimated"`, `"policy_checking"`, `"policy_override"`, `"policy_soft_failed"`, `"policy_checked"`, and `"planned_and_finished"`.
```
Comment (nfagerlund) [inline content/source/docs/cloud/api/admin/runs.html.md]:
```suggestion
`filter[status]`    | **Optional.** A comma-separated list of Run statuses to restrict results to, which can include any of the following: `"pending"`, `"plan_queued"`, `"planning"`, `"planned"`, `"confirmed"`, `"apply_queued"`, `"applying"`, `"applied"`, `"discarded"`, `"errored"`, `"canceled"`, `"cost_estimating"`, `"cost_estimated"`, `"policy_checking"`, `"policy_override"`, `"policy_soft_failed"`, `"policy_checked"`, and `"planned_and_finished"`.
```

**Comment by rberlind** [comment]
Thanks

**Pull request merged**